src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/HmppsRestrictedPatientsApiExceptionHandler.kt

```diff
@@ -190,17 +193,19 @@ class HmppsRestrictedPatientsApiExceptionHandler {
 
 data class ErrorResponse(
   val status: Int,
-  val errorCode: Int? = null,
+  val errorCode: String? = null,
```

Comment:
OK to change as not currently being used. Much prefer a descriptive String to some random Int.